android/BOINC/app/src/main/java/edu/berkeley/boinc/BOINCActivity.java

```diff
@@ -508,7 +511,7 @@ protected Boolean doInBackground(Integer... params) {
 
         @Override
         protected void onPostExecute(Boolean success) {
-            if(success) {
+            if(Boolean.TRUE.equals(success)) {
```

Comment:
Is there a better way to compare Boolean value? Maybe `Boolean` type could be changed to `boolean`?

Reply:
Java's generics system doesn't support the primitive types yet:

[JEP 218](http://openjdk.java.net/jeps/218)